packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -15,7 +15,9 @@ import {
 } from '../../utils/constants';
 
 export const DEFAULT_CONFIG_PER_VIEW_TYPE = {
-	[ LAYOUT_TABLE ]: {},
+	[ LAYOUT_TABLE ]: {
+		primaryField: 'title',
```

Comment:
The table layout wasn't defining what's the primary field. This is necessary for the primary field class to be serialized into the HTML.